rpc/src/v1/traits/eth.rs

```diff
@@ -235,6 +239,7 @@ pub trait EthSigning: Sized + Send + Sync + 'static {
 		delegate.add_method("eth_postSign", EthSigning::post_sign);
 		delegate.add_method("eth_postTransaction", EthSigning::post_transaction);
 		delegate.add_method("eth_checkRequest", EthSigning::check_request);
+		delegate.add_method("ethcore_decryptMessage", EthSigning::decrypt_message);
```

Comment:
Mixing of module names here? Strange that `EthSigning` would bridge both `eth` and `ethcore`.